.github/workflows/tests.yml

```diff
@@ -67,6 +51,17 @@ jobs:
         run: |
           redis-server /etc/redis/redis.conf &
 
+      - name: Start Postgres
+        run: |
+          chown -R postgres /var/run/postgresql
+          sudo -u postgres /usr/lib/postgresql/${{ matrix.postgres }}/bin/initdb -D /tmp/test_data/pg
+          echo fsync = off >> /tmp/test_data/pg/postgresql.conf
+          echo full_page_writes = off >> /tmp/test_data/pg/postgresql.conf
+          echo shared_buffers = 500MB >> /tmp/test_data/pg/postgresql.conf
+          sudo -u postgres /usr/lib/postgresql/13/bin/pg_ctl -D /tmp/test_data/pg start
+          sudo -u postgres /usr/lib/postgresql/13/bin/createuser -s $PGUSER
```

Comment:
You're using both `${{ matrix.postgres }}` and `13`. Should choose one of them. And ideally, can we make it use the latest pg available in the image? So that this wont break/need an update when we drop 13 from the image?

Reply:
> This commit also sets a few Postgres flags to improve test performance. We use these flags in our rake docker:test task already (i.e. our internal CI system).

Maybe we should provide this as a script in the image?

Reply:
Good idea, both done 👍